codegen/src/main/java/software/amazon/awssdk/codegen/model/intermediate/MemberModel.java

```diff
@@ -385,6 +385,30 @@ public String getFluentSetterDocumentation() {
         return docBuilder.toString();
     }
 
+    public String getDefaultConsumerFluentSetterDocumentation() {
+        return new StringBuilder()
+            .append(StringUtils.isNotBlank(documentation) ? documentation : DEFAULT_SETTER.replace("%s", name) + "\n")
+            .append(LF)
+            .append("This is a convenience which creates an instance of the {@link ")
```

Comment:
Can we just use string appending instead of the builder?

Reply:
could do - but this is how the other doc builders work...

Reply:
done.